### PR TITLE
Hw111

### PR DESCRIPTION
**Pull request by RomanPendyur**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/indexStyles.css">
+    <link rel="stylesheet" href="./styles/blogStyles.css">
+    <title>Document</title>
+</head>
+<body class="body__style">
```
Comment (roskyba) [inline index.html:11]: Poor class name - omit HTML elements names in classes

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <div class="grand_container__style">
+        <h1 class="h1_grand_container__style">Willow Crest</h1>
```
Comment (roskyba) [inline index.html:13]: and here as well. Widely spread mistakes. Please, re-check your code and rename all bad-named classes. Thanks

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+           
+           The travellers gazed on this curious spectacle from the platforms; but Phileas Fogg, who had the most reason of all to be in a hurry, remained in his seat,  and waited philosophically until it should please the buffaloes to get out of the way.
+           Passepartout was furious at the delay they occasioned, and longed to discharge his arsenal of revolvers upon them.<br><br>
+           
+           "What a country!" cried he. "Mere cattle stop the trains, and go by in a procession, just as if they were not impeding travel! Parbleu! I should like to know if Mr. Fogg foresaw this mishap in his programme! And here's an engineer who doesn't dare to run the locomotive into this herd of beasts!"<br><br>
+           <span class="labki_grabd_container__style">“ </span>
+           <span class="footer_grabd_container__style"> the engineer did not try to overcome the obstacle, and he was wise.</span><br><br>
+           </p>
+        <button class="button_grand_container__style"><span class="button_span_grand_container__style">DISCOVER</span></button>
```
Comment (roskyba) [inline index.html:24]: `button ===> btn` for example

File: styles/indexStyles.css [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    font-style: normal;
+    letter-spacing: normal;
+    line-height: 34px;
+    text-align: center;
+    text-transform: uppercase;
+    font-style: normal;
+    letter-spacing: 3.6px;
+    line-height: normal;
+}
+
```
Comment (roskyba) [inline styles/indexStyles.css:93]: remove unused extra new lines

**Comment by RomanPendyur** [comment]
Ok, I will try no repeat these errors.

**Pull request closed without merging**